pkgs/tools/graphics/gmic-qt/default.nix

```diff
@@ -54,13 +54,21 @@ mkDerivation rec {
   pname = "gmic-qt${lib.optionalString (variant != "standalone") "-${variant}"}";
   version = "3.2.1";
 
-  src = fetchFromGitHub {
-    owner = "c-koi";
-    repo = "gmic-qt";
-    rev = "v.${version}";
-    sha256 = "sha256-z+GtYLBcHVufXwdeSd8WKmPmU1+/EKMv26kNaEgyt5w=";
+  src = fetchzip {
+    url = "https://gmic.eu/files/source/gmic_${version}.tar.gz";
```

Comment:
IIRC, I switched to git because the tarball was missing some CMake files for `gmic` and I wanted to ensure that system GMic and CImg is actually used. Though that could be also achieved by removing the appropriate directories in `postPatch`.

Reply:
Yep the tarball is missing the gmic CMake files, but that isn't a problem for gmic-qt so I took `gmic-qt` (but only that one) back from git to tarball

With the gmic-qt PR, this does correctly link against the system libgmic and does not build the static lib or anything like that